__tests__/app.test.js

```diff
@@ -32,24 +32,59 @@ describe('GET /api/topics', () => {
       .then(({ body: { topics } }) => {
         expect(topics.length).toBe(3);
         topics.forEach((topic) => {
-          expect(topic).toEqual(
-            expect.objectContaining({
-              slug: expect.any(String),
-              description: expect.any(String),
-            })
-          );
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
         });
       });
   });
 });
 
-describe('Error handling: non-existent route queried', () => {
+describe('GET /api/articles/:article_id', () => {
+  test('200: Responds with the article linked to :article_id', () => {
+    return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toEqual({
+          article_id: 1,
+          title: 'Living in the shadow of a great man',
+          topic: 'mitch',
+          author: 'butter_bridge',
+          body: 'I find this existence challenging',
+          created_at: '2020-07-09T20:11:00.000Z',
+          votes: 100,
+          article_img_url:
+            'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+        });
+      });
+  });
+  test('400: Responds with an error message if article_id is not a number', () => {
+    return request(app)
+      .get('/api/articles/soup')
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe(`Article ID must be a number.`);
+      });
+  });
+  test('400: Responds with an error message if article_id does not exist', () => {
+    return request(app)
+      .get('/api/articles/99999')
+      .expect(400)
```

Comment:
we have tested for a bad request above, you would want to consider other status codes as feedback to handle some other errors.